hagrid/reservations/export_pdf.py

```diff
@@ -293,14 +309,23 @@ def render_invoice_end(l, d: Document):
 
 
 def render_reservation(r: Reservation, d: Document):
-    if r.state == Reservation.STATE_SUBMITTED:
+    if r.state == r.STATE_SUBMITTED:
```

Comment:
```suggestion
    if r.state == Reservation.STATE_SUBMITTED:
```

Reply:
I don't know. Referencing static constants should be independent of local objects.

Reply:
It's a style thing I guess, but I'd like to make clear that is a constant related to the class, not the object.

Reply:
Oh I just saw that I was the one who wrote `r.STATE_SUBMITTED`. Well I thought you suggested me to use `r` instead of `Reservation` which staged my protest. That mistake will be fixed soon.